packages/components/inputs/rich-text-utils/src/localized/localized.ts

```diff
@@ -0,0 +1,64 @@
+import { warning } from '@commercetools-uikit/utils';
+import uniq from 'lodash/uniq';
+import html from '../html';
+import isRichTextEmpty from '../is-empty';
+
+type TLocalizedString = {
+  language: string;
+};
+
+type TTranslations = {
```

Comment:
I've seen that we're using `Record<string, string>` is other parts of the repository for this case:
```
type TTranslations = Record<string, string>;
```